sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -103,7 +103,7 @@ class CacheManager extends Logging {
   def uncacheQuery(
       query: Dataset[_],
       cascade: Boolean,
-      blocking: Boolean = true): Unit = {
```

Comment:
Why even keep the parameter? This is internal to Spark right?

Reply:
Dropped it

Reply:
I'm checking now and just a sec.

Reply:
ok, removed it.